static/operator-profile-default.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: install.istio.io/v1alpha2
```

Comment:
we can do this for now if we want, but long term should this just redirect to somewhere where we store release artifacts (GCS)? That way we don't need to update this manually/semi-automatically